### PR TITLE
V22F-496 save files to temp

### PR DESCRIPTION
**Pull request by KungFuDonkey**
Moved most of the create dirs to the users Appdata/local/temp folder and moved the recordings to Appdata/Roaming/sda

### CODE REVIEW

File: speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/CMakeLists.txt [hunk 1]
```diff
@@ -79,4 +79,8 @@ ADD_MYSQL_PACKAGE(backend)
 
 ADD_BOOST_PACKAGE(backend)
 
-SD_INSTALL_FILES(LIB simulatedDrivingAssistance/backend TARGETS backend)
\ No newline at end of file
+SD_INSTALL_FILES(LIB simulatedDrivingAssistance/backend TARGETS backend)
+
+MESSAGE(STATUS "BANANEN ${CMAKE_INSTALL_PREFIX}")
```
Comment (suppergerrie2) [inline speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/CMakeLists.txt:84]:
```suggestion
```

Comment (larsdekwant) [inline speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/CMakeLists.txt]: :)

File: speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Recorder.cpp [hunk 2]
```diff
@@ -16,10 +16,6 @@ Recorder::Recorder(const std::string& p_dirName, const std::string& p_fileName,
 {
     // create directory if it doesn't exist
     std::string dirPath = "..\\test_data\\" + p_dirName;
-    if (!GfDirExists(dirPath.c_str()))
-    {
-        GfDirCreate(dirPath.c_str());
-    }
```
Comment (suppergerrie2) [inline speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Recorder.cpp:22]: It still needs to create the directory right? It just needs a different location

**Review by suppergerrie2 — APPROVED** [review]

File: speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Recorder.cpp (hunk 2 above)
Comment (KungFuDonkey) [reply speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Recorder.cpp:22]: No it is created by cmake now

**Review by suppergerrie2 — CHANGES_REQUESTED** [review]
Accidentally approved

File: speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Recorder.cpp (hunk 2 above)
Comment (suppergerrie2) [reply speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Recorder.cpp:22]: Also when you install it?

**Review by larsdekwant — CHANGES_REQUESTED** [review]
I also like BANANEN, but not in my CMake

File: speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Recorder.cpp (hunk 2 above)
Comment (KungFuDonkey) [reply speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Recorder.cpp]: yes, but it has been changed now

**Review by suppergerrie2 — APPROVED** [review]

**Review by larsdekwant — APPROVED** [review]

**Pull request merged**